tellerlogger/src/main/java/com/kiliansteenman/teller/logger/ui/overview/TellerLogActivity.kt

```diff
@@ -74,3 +107,9 @@ internal class TellerLogActivity : AppCompatActivity(R.layout.teller) {
         return true
     }
 }
+
+internal sealed class OverViewState {
+    object Loading : OverViewState()
+    data class Content(val logs: List<TellerLog>) : OverViewState()
+    data class Error(@StringRes val message: Int) : OverViewState()
```

Comment:
Nitpick: You could simplify this to just `object Empty: OverViewState()` and give the view (xml) the responsibility to show the message since it seems the only 'error message' we display 🤷

Reply:
Wanted to display different error messages according to different types. But might indeed just get rid of it for now.